modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/MultiSearchTemplateResponse.java

```diff
@@ -143,11 +170,15 @@ public void writeTo(StreamOutput out) throws IOException {
         for (Item item : items) {
             item.writeTo(out);
         }
+        if (out.getVersion().onOrAfter(Version.V_7_0_0_alpha1)) {
+            out.writeVLong(tookInMillis);
+        }
     }
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, ToXContent.Params params) throws IOException {
         builder.startObject();
+        builder.field("took", tookInMillis);         
```

Comment:
I guess this should have been added as part of #23767 . Can we make this change in a separate PR please? That way it will have a bit more visibility?

Reply:
OK - that issue/PR will have to be pushed before this one because the fromXContent part of this PR was failing due to the missing `tookInMillis` property.